### PR TITLE
4 get/api/articles/ :article

### PR DESCRIPTION
**Pull request by h-chagas**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,37 +1,73 @@
...
+            response.body.forEach((topic) => {
+               expect(topic).toMatchObject({
+                  slug: expect.any(String),
+                  description: expect.any(String),
+               });
+            });
+         });
+   });
+});
+describe.only('GET /api/articles/:article_id', () => {
```
Comment (yalicee) [inline __tests__/app.test.js]: remove .only please

Comment (h-chagas) [reply __tests__/app.test.js:36]: sorry

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,37 +1,73 @@
...
+describe('GET /api/articles/:article_id', () => { 
+    test('200: responds with an article object corresponded with article number passed', () => {
+        const ARTICLE_ID = 1;
+        return request(app)
+        .get(`/api/articles/${ARTICLE_ID}`)
+        .expect(200)
+        .then((response) => {
+            expect(typeof response.body).toBe('object')
+            expect(response.body).toMatchObject({
+                article_id: expect.any(Number),
```
Comment (yalicee) [inline __tests__/app.test.js]: The expected keys have correct values - must be values related to the requested article_id, e.g. NOT article_id: expect.any(Number)

**Review by yalicee — APPROVED** [review]
See my comment and change tests before you merge. Otherwise it is approved so no need to send in another PR :)

**Pull request merged**